### PR TITLE
Introduce NotifyEmailBuilder

### PR DESCRIPTION
**Pull request by nupplaphil**
FollowUp #8215 and #8176 

Introduces a `NotifyEmailBuilder` as another type of `MailBuilder`

I test some notifications on my node before I make it ready for merging :)

### CODE REVIEW

File: src/Util/Emailer.php [hunk 1]
```diff
@@ -9,9 +9,11 @@
 use Friendica\Core\Hook;
 use Friendica\Core\L10n;
 use Friendica\Core\PConfig\IPConfig;
+use Friendica\Model\Notify;
```
Comment (MrPetovan) [inline src/Util/Emailer.php]: Please remove this if it's only used in a doc block.

File: view/templates/email/notify/html.tpl [hunk 2]
```diff
@@ -32,6 +17,4 @@
 		<tr><td></td><td>{{$thanks}}</td></tr>
 		<tr><td></td><td>{{$site_admin}}</td></tr>
 	</tbody>
-	</table>
-</body>
-</html>
+	</table>
```
Comment (MrPetovan) [inline view/templates/email/notify/html.tpl]: Please add an EOF EOL.

**Comment by nupplaphil** [comment]
Works :) Got a full notify with content / without content and a "reset password" mail ..

**Comment by MrPetovan** [comment]
20 hours later, still behaving like expected?

**Comment by nupplaphil** [comment]
I git about 10-15 notifications, nothing problematic detected.. The mail style is a bit different due to the split into two tables for banner and content, but almost equal.

**Pull request merged**